docs/README.md

```diff
@@ -185,16 +185,38 @@ button {
 ```
 
 ## Reset
-Nib comes bundled with Eric Meyer's style reset support, you can choose to apply the global or any specifics that you wish. To view the definitions view [reset.styl](https://github.com/visionmedia/nib/blob/master/lib/nib/reset.styl)
-
-- global-reset()
-- nested-reset()
-- reset-font()
-- reset-box-model()
-- reset-body()
-- reset-table()
-- reset-table-cell()
-- reset-html5()
+Nib comes bundled with [Eric Meyer's style reset](eric-meyer) and [Nicolas Gallagher's _Normalize_](normalize) support and, you can choose to apply the global or any specifics that you wish. To view the definitions view [`reset.styl`](https://github.com/visionmedia/nib/blob/master/lib/nib/reset.styl).
+
+[eric-meyer]: http://meyerweb.com/eric/tools/css/reset/
+[normalize]: https://github.com/necolas/normalize.css
+
+> CSS Reset
+
+- `global-reset()`
+- `nested-reset()`
+- `reset-font()`
+- `reset-box-model()`
+- `reset-body()`
+- `reset-table()`
+- `reset-table-cell()`
+- `reset-html5()`
+
+> Normalize
+
+- `normalize-html5()`
+- `normalize-base()`
+- `normalize-links()`
+- `normalize-text()`
+- `normalize-embed()`
+- `normalize-groups()`
+- `normalize-forms()`
+- `normalize-tables()`
+- `normalize-all()`
```

Comment:
We might be better off calling this `normalize-css()`. In [axis](http://axis.bitballoon.com) (which is built atop nib), that's what it's called. So using the same name might make updating a bit easier for the axis folks.